gallery/src/components/NxTabs/NxTabsPage.tsx

```diff
@@ -65,6 +71,16 @@ export default function NxTabsPage() {
                   Called with the index of the newly selected tab when the currently selected tab changes.
                 </NxTableCell>
               </NxTableRow>
+              <NxTableRow>
+                <NxTableCell>.nx-modal-content--tabs</NxTableCell>
```

Comment:
This is the table of _props_ for `NxTabs`, so it's really not the right place for this class to be documented.  I'd document it on the `NxModal` page (in the classes table), though I think it's alright to still demonstrate it here.

Reply:
> This is the table of _props_ for `NxTabs`, so it's really not the right place for this class to be documented. I'd document it on the `NxModal` page (in the classes table), though I think it's alright to still demonstrate it here.

Changed here: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/07abe89ddbbeb360e4b1b948735fd7eee23402a2 and text tweaked here: https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/6efdf8bd3af9065bdb7e501109679b1a4651d600